can-component.js

```diff
@@ -253,12 +269,63 @@ var Component = Construct.extend(
 					options.tags = {};
 				}
 
+				options.parentRenderer = this.parentRenderer;
+				options.parentViewModel = this.viewModel;
+				options.parentTag = this.parentTag;
+
+				options.tags.super = function superHookup(el, superTagData) {
```

Comment:
is it possible to make this generic so the same code isn't duplicated for this and `<content>`?  I know the `<slot>` folks will want similar stuff.  Maybe everyone can work together on that @mikemitchel @Sinjhin